backend/src/controllers/game_controller.ts

```diff
@@ -67,7 +67,7 @@ export class GameController extends Controller {
 
     // save new game
     const game: Game = new Game();
-    game.startDate = new Date();
+    game.startDate = new Date(0);
```

Comment:
Why do you need to set this to the first date ?

Reply:
I want to be able to tell if the game has started or not with its `startDate`. Setting it with a default value of 0 (which corresponds to the Unix epoch, 01/01/1970) allows us to make the difference between a game that has started or not.